docs/source/quickstart.md

```diff
@@ -14,235 +18,193 @@ pip install mammoth-nlp
+## Experiment 2: Machine translation with multi30k
 
-## Step 3: Start training
+### Step 1: Set the locations
 
-You can start your training on a single machine, by simply running a python script `train.py`, possibly with a definition of your desired GPUs. 
-Note that the example config above assumes two GPUs available on one machine.
+Set the locations to the directory in which you want to work on this project, and the path to the mammoth git repo.
 
-```shell
-CUDA_VISIBLE_DEVICES=0,1 python3 train.py -config my_config.yaml -save_model output_dir -tensorboard -tensorboard_log_dir log_dir
+```bash
+export PROJECT_DIR="/path/to/work/dir"
+export MAMMOTH_DIR="/path/to/mammoth"
 ```
 
-Note that when running `train.py`, you can use all the parameters from [train.py](options/train) as cmd arguments. In the case of duplicate arguments, the cmd parameters override the ones found in your config.yaml.
+### Step 2: Activate your virtual env
 
+```bash
+source ~/venvs/mammoth/bin/activate
+```
 
+### Step 3: Download data
 
-### Step 4: Translate
+```bash
+cd $PROJECT_DIR
+mkdir data/multi30k
+pushd data/multi30k
+
+for language in cs en de fr; do
+    wget "https://github.com/multi30k/dataset/raw/refs/heads/master/data/task1/raw/test_2016_flickr.${language}.gz"
+    wget "https://github.com/multi30k/dataset/raw/refs/heads/master/data/task1/raw/val.${language}.gz"
+    wget "https://github.com/multi30k/dataset/raw/refs/heads/master/data/task1/raw/train.${language}.gz"
+done
+popd
+```
 
-Now that you have successfully trained your multilingual machine translation model using Mammoth, it's time to put it to use for translation. 
+### Step 4: Train sentencepiece models
 
 ```bash
-python3 -u $MAMMOTH/translate.py \
-  --config "my_config.yml" \
-  --model "$model_checkpoint" \
-  --task_id  "train_$src_lang-$tgt_lang" \
-  --src "$path_to_src_language/$lang_pair.$src_lang.sp" \
-  --output "$out_path/$src_lang-$tgt_lang.hyp.sp" \
-  --gpu 0 --shard_size 0 \
-  --batch_size 512
+mkdir -p models/spm
+for language in cs en de fr; do
+    zcat data/multi30k/train.${language}.gz > /tmp/spm_train_${language}.txt
+    spm_train --input /tmp/spm_train_${language}.txt --model_prefix=models/spm/spm.${language} --vocab_size 8000
+    rm /tmp/spm_train_${language}.txt
+done
 ```
 
-Follow these configs to translate text with your trained model.
+### Step 5: Copy the config template from the Mammoth repo
 
-- Provide necessary details using the following options:
-   - Configuration File: `--config "my_config.yml"`
-   - Model Checkpoint: `--model "$model_checkpoint"`
-   - Translation Task: `--task_id "train_$src_lang-$tgt_lang"`
+```bash
+mkdir config
+cp -i ${MAMMOTH_DIR}/examples/multi30k.template.yaml config/multi30k.template.yaml
+```
+
+### Step 6: Generate the actual config from the config template 
+
+(this should only take a few seconds)
+
+```bash
+python ${MAMMOTH_DIR}/tools/config_config.py \
```

Comment:
use mammoth tool

Reply:
Fixed in https://github.com/Helsinki-NLP/mammoth/pull/86